app/helpers/permissions_helper.rb

```diff
@@ -21,7 +21,7 @@ def permissions_control_class(permissionable)
   end
 
   def permissions_switch_class(permissionable)
-    permissionable.class == Permission ? 'permission-switch' : 'master'
+    permissionable.class == Permission ? "permission-switch" : "master"
```

Comment:
If there are several types of permissionable, we could have several presenters and move this logic there. Same here and in `permissions_check_box_value` method.